docs-src/concepts/what-is-a-memo.md

```diff
@@ -2,10 +2,29 @@
 id: what-is-a-memo
 title: What is a Memo?
 sidebar_label: Memo
-description: A Memo is a non-indexed user-supplied set of Workflow Execution metadata that is displayed with Filtered List results.
+description: A Memo is a non-indexed user-supplied set of Workflow Execution metadata that is displayed with filtered list results.
 tags:
   - term
   - explanation
 ---
 
-A Memo is a non-indexed user-supplied set of Workflow Execution metadata that is displayed with Filtered List results.
+A Memo is a non-indexed set of Workflow Execution metadata that users supply and that a filtered list displays.
+Use Memos to define metadata supplied by users in a filtered list.
+
+The primary purpose of using a Memo is to enhance the organization and management of Workflow Executions.
+Add your own metadata, such as notes or descriptions, to a Workflow Execution, which lets you annotate and categorize Workflow Executions based on user-defined criteria.
```

Comment:
```suggestion
A Memo is a non-indexed set of Workflow Execution metadata that you supply at start time or in Workflow code and that is returned when you describe or list Workflow Executions.

The primary purpose of using a Memo is to enhance the organization and management of Workflow Executions.
Add your own metadata, such as notes or descriptions, to a Workflow Execution, which lets you annotate and categorize Workflow Executions based on criteria you define.
```

- clarify user = developer / the reader
- the two API calls that return memos are describe and list

Reply:
execute/start is the "at start time". the returned values, in describe and list, might be --fields in the CLI commands. it's also in the gRPC API and SDK APIs.